<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="scanButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>54</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_feed">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_grip">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="linkButton_grip">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Link Grip</string>
     </property>
    </widget>
    <widget class="QPushButton" name="linkButton_feed">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Link Feed</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_release">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_posP">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_posI">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_posD">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_currP">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_currD">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_currI">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Curr</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>49</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>49</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>49</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
